scripts/cray_runall.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+alps_present=`which alps`
+if [ $? == 0 ]; then
+launcher="aprun -n 1"
+else
+launcher="srun -n 1"
+fi
+echo Running fi_info
+$launcher ./fi_info 2>&1 | tee test.output
```

Comment:
I think I'd prefer this script to either *cd* to the proper directory (*simple*, in this case) or assume the script is being run from the top-level (i.e., *./simple/fi_info*).